permissions/plugin-aws-codecommit-trigger.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "aws-codecommit-trigger"
+paths:
+  - "org/jenkins-ci/plugins/aws-codecommit-trigger"
```

Comment:
@phuonghuynh Thanks, I'll wait for @ronaldtse to fix his account email address before merging.